### ISSUE TITLE
Paths in pkgconfig file are expanded twice

### ISSUE THREAD

**Issue by e-kwsm** [jbeder/yaml-cpp#431]
### Problem

I installed yaml-cpp to custom location like
```
$ git clone https://github.com/jbeder/yaml-cpp
$ mkdir yaml-cpp/build
$ cd yaml-cpp/build
$ cmake .. -DCMAKE_INSTALL_PREFIX=/tmp/local/yaml-cpp
$ make install
```

But the generated pkgconfig file (`/tmp/local/yaml-cpp/lib/pkgconfig/yaml-cpp.pc` in this case) is
```
prefix=/tmp/local/yaml-cpp
exec_prefix=/tmp/local/yaml-cpp
libdir=${prefix}//tmp/local/yaml-cpp/lib
includedir=${prefix}//tmp/local/yaml-cpp/include

Name: Yaml-cpp
Description: A YAML parser and emitter for C++
Version: 0.5.3
Requires:
Libs: -L${libdir} -lyaml-cpp
Cflags: -I${includedir} 
```
`libdir` and `includedir` have `${prefix}` and __already expanded__ paths; so `Cflags` becomes `-I/tmp/local/yaml-cpp//tmp/local/yaml-cpp/include`.
The file should be
```
libdir=${prefix}/lib
includedir=${prefix}/include
```

### Environment

- OS: Debian Jessie
- Cmake: version 3.0.2
- pkg-config: version 0.28

**Comment by ovidiub13** [comment]
I cannot replicate this with:

* yaml-cpp `0.5.3`
* CMake `3.5.2`
* Kubuntu `16.10`

```
prefix=/home/ovidiu/WORK/YAML Test/Install
exec_prefix=/home/ovidiu/WORK/YAML Test/Install
libdir=${prefix}/lib
includedir=${prefix}/include
```

@e-kwsm what version of yaml-cpp did you use?

**Comment by e-kwsm** [comment]
@ovidiub13 Thank you for your reply.

> what version of yaml-cpp did you use?

The latest version of the github repository when I raised this issue; and I can reproduce now (master 380ecb4)

`yaml-cpp.pc.cmake` is
```
     1	prefix=@CMAKE_INSTALL_PREFIX@
     2	exec_prefix=@CMAKE_INSTALL_PREFIX@
     3	libdir=${prefix}/@LIB_INSTALL_DIR@
     4	includedir=${prefix}/@INCLUDE_INSTALL_ROOT_DIR@
```
and the setting of the variables in `CMakeLists.txt` is
```
   265	set(INCLUDE_INSTALL_ROOT_DIR ${CMAKE_INSTALL_PREFIX}/include)
   268	set(LIB_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/${_library_dir}${LIB_SUFFIX}")
```

I am not familiar with CMake, but it  seems to expand `CMAKE_INSTALL_PREFIX` twice.

**Comment by jbeder** [comment]
Fixed now?

**Comment by e-kwsm** [comment]
Yes, fixed (master d025040). Thank you.
